### PR TITLE
Fixed attribute step contextualisation

### PR DESCRIPTION
**Pull request by rousso**
I discovered this while I was investigating another bug. Apparently we have been doing this wrong all along and we never noticed.
Can you verify that my fix renders the proper results. Please look at the modified test cases.
Do not merge to develop yet. Only approve or reject or comment please.

### CODE REVIEW

**Review by YvesJo — COMMENTED** [review]
Can not express any opinion

File: src/test/java/eu/europa/ted/efx/XPathContextualizerTest.java [hunk 1]
```diff
@@ -60,12 +60,12 @@ void testAttributeInXpath() {
 
   @Test
   void testAttributeInContext() {
-    assertEquals("../c/d", contextualize("/a/b/@attribute", "/a/b/c/d"));
+    assertEquals("c/d", contextualize("/a/b/@attribute", "/a/b/c/d"));
```
Comment (bertrand-lorentz) [inline src/test/java/eu/europa/ted/efx/XPathContextualizerTest.java:63]: This does not seem correct: an attribute is like a child. So the ../ is needed.
See this example: http://xpather.com/ZjCLbf5X

File: src/test/java/eu/europa/ted/efx/XPathContextualizerTest.java [hunk 2]
```diff
@@ -60,12 +60,12 @@ void testAttributeInXpath() {
...
   }
 
   @Test
   void testAttributeInBoth() {
-    assertEquals("../@x", contextualize("/a/b/c/@d", "/a/b/c/@x"));
+    assertEquals("@x", contextualize("/a/b/c/@d", "/a/b/c/@x"));
```
Comment (bertrand-lorentz) [inline src/test/java/eu/europa/ted/efx/XPathContextualizerTest.java:68]: Same issue as above, see the following example:
http://xpather.com/MJgKQ38D

**Comment by rousso** [comment]
Not for merge. I will remove this branch

**Pull request closed without merging**